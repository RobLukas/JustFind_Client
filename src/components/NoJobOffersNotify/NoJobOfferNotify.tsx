import React, { FC } from 'react';
import S from './NoJobOfferNotify.styles';

const NoJobOfferTitle: FC = () => {
  return (
    <S.Wrapper>
      <S.Title>{'Sorry, there are no job offers.'}</S.Title>
    </S.Wrapper>
  );
};

export default NoJobOfferTitle;
