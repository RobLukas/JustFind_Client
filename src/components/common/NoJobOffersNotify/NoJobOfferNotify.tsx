import React, { FC } from 'react';
import S from './NoJobOfferNotify.styles';

const NoJobOfferNotify: FC = () => {
  return (
    <S.Wrapper>
      <S.Title>{'Sorry, there are no job offers.'}</S.Title>
    </S.Wrapper>
  );
};

export default NoJobOfferNotify;
