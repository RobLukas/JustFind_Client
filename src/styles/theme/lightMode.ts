export default {
  name: 'Light Theme',
  styles: {
    primary: {
      background: '#f3f6f8',
      color: '#37474f',
    },
    secondary: {
      background: '#fff',
      color: '#99a1ab',
    },
    dropdown: {
      button: {
        hover: '#f3f6f8',
      },
    },
    scrollbar: {
      thumbBackground: '#c1c1c1',
      background: 'rgba(0,0,0,0.07)',
      hover: 'rgba(0,0,0,0.09)',
    },
    filter: {
      button: {
        color: '#777',
      },
      selected: {
        background: 'linear-gradient(to right, #ba68c8, #9c27b0)',
        color: '#fff',
      },
    },
    offer: {
      header: {
        title: '#fff',
        filter: {
          brightness: '1',
        },
        icons: {
          companyName: '#ff5252',
          companySize: '#fb8c00',
          companyEMPType: '#ab47bc',
          companySkill: '#66bb6a',
          new: '#448aff',
        },
      },
      desc: {
        seperateLine: {
          color: '#f3f6f8',
        },
        skillPoint: {
          selected: '#ff4081',
          unselected: '#393939',
        },
      },
    },
    offers: {
      newTitle: {
        background: '#d9ddfc',
        color: '#4256ef',
      },
      techTitle: {
        background: '#fff',
        border: '#e0e0e0',
      },
    },
  },
};
