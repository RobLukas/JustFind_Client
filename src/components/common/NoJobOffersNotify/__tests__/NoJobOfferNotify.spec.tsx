import React from 'react';
import { cleanup, render } from '@testing-library/react';
import 'jest-styled-components';
import { ThemeProvider } from 'styled-components';

import NoJobOfferNotify from '../NoJobOfferNotify';
import lightMode from '@styles/theme/lightMode';

describe('<NoJobOfferNotify />', () => {
  afterEach(cleanup);
  it('render correctly', () => {
    const { container } = render(
      <ThemeProvider theme={lightMode}>
        <NoJobOfferNotify />
      </ThemeProvider>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
