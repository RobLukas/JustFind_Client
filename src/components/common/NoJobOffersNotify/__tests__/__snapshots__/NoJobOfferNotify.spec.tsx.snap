// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoJobOfferNotify /> render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: absolute;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  font-size: 14;
  left: 0;
  top: 0;
  height: 70;
  width: 100%;
}

.c1 {
  color: #99a1ab;
}

<div
  class="c0"
>
  <span
    class="c1"
  >
    Sorry, there are no job offers.
  </span>
</div>
`;
